src/quo2/components/list_items/user_list.cljs

```diff
@@ -49,8 +48,8 @@
      :profile-picture photo-path
      :online?         online?
      :size            :small}]
-   [rn/view {:style {:margin-left 8}}
-    [author/author
+   [rn/view {:style {:margin-horizontal 8 :flex 1}}
```

Comment:
Should this be in a style.cljs file?

Reply:
The current implementation of user-list component is not using the view, style file structure. I have created a separate issue to track the refactor: https://github.com/status-im/status-mobile/issues/17415